airbyte-cdk/java/airbyte-cdk/db-destinations/build.gradle

```diff
@@ -1,93 +1,28 @@
-
 java {
+    // TODO: rewrite code to avoid javac wornings in the first place
     compileJava {
-        options.compilerArgs += "-Xlint:-deprecation"
+        options.compilerArgs += "-Xlint:-deprecation,-removal"
+    }
+    compileTestFixturesJava {
+        options.compilerArgs += "-Xlint:-try"
     }
 }
 
 dependencies {
-    // Depends on core CDK classes (OK 👍)
-    implementation project(':airbyte-cdk:java:airbyte-cdk:core')
-    testFixturesImplementation project(':airbyte-cdk:java:airbyte-cdk:typing-deduping')
-    testFixturesImplementation testFixtures(project(':airbyte-cdk:java:airbyte-cdk:typing-deduping'))
+    api 'org.apache.commons:commons-csv:1.10.0'
```

Comment:
CSVs are everywhere anywhere we import this module so I thought we'd be better off defining this as an `api` rather than an `implementation` dependency.